rpcd/playbooks/roles/rpc_pre_upgrade/tasks/main.yml

```diff
@@ -25,6 +25,7 @@
 
 # Gather swift-recon md5 output
 - include: gather_swift_stats.yml
+  when: groups['swift_proxy']|default([])|length > 0
```

Comment:
The pre-upgrade Swift task have the condition
```
when: inventory_hostname in groups['swift_proxy']
```
In general there seems to be a lot of overlap between the Swift bits of the pre and post roles, not suggesting that this patch should address that but maybe we should be consistent between the two. Is one condition better than the other?